### PR TITLE
AHOYAPPS-819 Add AudioSwitch version constant

### PR DESCRIPTION
**Pull request by aaalaniz**
## Description

Add AudioSwitch version constant

## Validation

- Unit and integration test

## Submission Checklist
 - [x] The source has been evaluated for semantic versioning changes and are reflected in `gradle.properties`
 - [x] The `CHANGELOG.md` reflects any **feature**, **bug fixes**, or **known issues** made in the source code

### CODE REVIEW

File: audioswitch/src/test/java/com/twilio/audioswitch/selection/AudioDeviceSelectorJavaTest.java [hunk 1]
```diff
@@ -92,6 +93,16 @@ public void shouldAllowSelectingAudioDevice() {
         assertEquals(earpiece, javaAudioDeviceSelector.getSelectedAudioDevice());
     }
 
+    @Test
+    public void getVersion_shouldReturnValidSemverFormattedVersion() {
+        String semVerRegex =
+                "^([0-9]+)\\.([0-9]+)\\.([0-9]+)(?:-([0-9A-"
+                        + "Za-z-]+(?:\\.[0-9A-Za-z-]+)*))?(?:\\+[0-9A-Za-z-]+)?$";
+
+        assertNotNull(AudioDeviceSelector.VERSION);
```
Comment (Alton09) [inline audioswitch/src/test/java/com/twilio/audioswitch/selection/AudioDeviceSelectorJavaTest.java:102]: Can we stick to using one assertion framework to be consistent? Most of the tests use the `org.hamcrest.CoreMatchers` class.

File: audioswitch/src/main/java/com/twilio/audioswitch/selection/AudioDeviceSelector.kt [hunk 2]
```diff
@@ -27,6 +28,13 @@ private const val TAG = "AudioDeviceSelector"
  */
 class AudioDeviceSelector {
 
+    companion object {
+        /**
+         * The version of the AudioSwitch library.
+         */
+        const val VERSION = BuildConfig.VERSION_NAME
```
Comment (Alton09) [inline audioswitch/src/main/java/com/twilio/audioswitch/selection/AudioDeviceSelector.kt:35]: I think we agreed on logging this at instantiation time right?

Comment (aaalaniz) [reply audioswitch/src/main/java/com/twilio/audioswitch/selection/AudioDeviceSelector.kt:35]: I didn't see that in the AC. I think we discussed adding a prefix to log messages that included the version.

Comment (Alton09) [reply audioswitch/src/main/java/com/twilio/audioswitch/selection/AudioDeviceSelector.kt:35]: Oh okay yep that works too. I'll include that in the logging story.

**Pull request merged**